go/cmd/vtorc/main.go

```diff
@@ -101,6 +102,7 @@ func main() {
 	// TODO(ajm188): after v15, remove this pflag hack and use servenv.ParseFlags
 	// directly.
 	fs := pflag.NewFlagSet("vtorc", pflag.ExitOnError)
+	vtlog.RegisterFlags(fs)
```

Comment:
@GuptaManan100 @rsajwani IIUC this should go away when we change vtorc to start using servenv.

Reply:
I would think yes, since serenv already has these flags registered

Reply:
yes, anyone that uses `servenv` to parse flags (WithArgs or not) doesn't need to do this manually